internal/cli/apis.go

```diff
@@ -249,7 +280,16 @@ auth0 apis update <id|audience> --name myapi`,
 				return err
 			}
 
-			api := &management.ResourceServer{}
+			if err := apiTokenLifetime.Ask(cmd, &inputs.TokenLifetime, nil); err != nil {
+				return err
+			}
+
+			apiAllowOfflineAccess.AskBool(cmd, &inputs.AllowOfflineAccess, nil)
```

Comment:
```suggestion
			apiAllowOfflineAccess.AskBoolU(cmd, &inputs.AllowOfflineAccess, nil)
```
Use `AskBoolU` for update command